documentation/experimental_features.md

```diff
@@ -216,6 +216,18 @@ ResourceInstance.new(
 )
 ```
 
+### Accessing `ResourceInstance` object on a Target
+
+You can get a specific `ResourceInstance` object stored on a Target in a plan using the `resource()`
+method:
```

Comment:
```suggestion
function:
```

Reply:
So pedantic. This is *technically* also a method on the target class.